### PR TITLE
[CONTRIB] Updates to MulticolumnValuesSumValuesEqualToSingleColumn

### PR DESCRIPTION
**Pull request by calvingdu**
Making functionality of this expectation to work on float values instead of integers to allow for more versatile check. Also added functionality for additional_value in the case of wanting to have another static numeric value or only using that numeric value

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by calvingdu** [comment]
@cla-bot

**Comment by calvingdu** [comment]
@cla-bot check

File: contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py [hunk 1]
```diff
@@ -85,12 +96,14 @@ class ExpectMulticolumnSumValuesToBeEqualToSingleColumn(MulticolumnMapExpectatio
     examples = [
         {
             "data": {
-                "col_a": [3, 6, 0, 1],
-                "col_b": [-6, -3, 1, 2],
-                "col_c": [1, 0, -1, 3],
-                "col_d": [-2, 3, 0, 6],
-                "col_e": [3, 6, 0, 1],
-                "col_f": [-3, 3, 1, 3],
```
Comment (anthonyburdi) [inline contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py:93]: What do you think about including a few test cases with integer values as well as floats to make sure the existing functionality still works?

File: contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py [hunk 2]
```diff
@@ -73,10 +81,13 @@ class ExpectMulticolumnSumValuesToBeEqualToSingleColumn(MulticolumnMapExpectatio
 
     Args:
         column_list (list of str): \
-            A list of 2 or more integer columns, in which we expect the sum of the first n-1th \
+            A list of 2 or more float columns, in which we expect the sum of the first n-1th \
```
Comment (anthonyburdi) [inline contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py]: It appears that integers would also work, how about changing to:
```suggestion
            A list of 2 or more int or float columns, in which we expect the sum of the first n-1th \
```

**Review by anthonyburdi — CHANGES_REQUESTED** [review]
Hi @calvingdu! Thank you for this contribution. I only have two small requests but otherwise this looks great!
Also I noticed that the CLA is showing as not signed, would you mind retrying to sign with the email address associated with your commits?:
Users missing a CLA: [calvindu@dhcp-206-12-66-221.ubcsecure.wireless.ubc.ca](mailto:calvindu@dhcp-206-12-66-221.ubcsecure.wireless.ubc.ca)

**Comment by calvingdu** [comment]
@cla-bot check

File: contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py (hunk 1 above)
Comment (calvingdu) [reply contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py:93]: Sure!

**Review by anthonyburdi — APPROVED** [review]
Thanks for the changes!

**Pull request merged**